### PR TITLE
Add bet form

### PR DESCRIPTION
**Pull request by ghost**
## What

Add bet form with validation. Confirming a bit should show in the developer tools' console:

```
Betting <amount> to <ticker>
```

## Pending

Add `prepend-icon="dollar"` to amount field.

## Screenshots

image
image
image

### CODE REVIEW

**Comment by Tommytrg** [comment]
You should be able to add icons with the changes in `master`

File: src/data/tickers.js [hunk 1]
```diff
@@ -0,0 +1,602 @@
+export default [
```
Comment (Tommytrg) [inline src/data/tickers.js:1]: I would reduce this list to 10 items

Comment (ghost) [reply src/data/tickers.js:1]: why don't we just leave it as it is, and when we know exactly how much we want, we leave those? Having all of them is useful, for example made me realize that if we have too much we need to use a select with an autocomplete

Comment (Tommytrg) [reply src/data/tickers.js:1]: I think @mariocao said that we are going list 10 assets but leave it as it is until we have the list

File: src/components/BetForm.vue [hunk 2]
```diff
@@ -1,12 +1,122 @@
...
+      resetData(this)
+    },
+
+    submit (event) {
+      this.confirmSubmission = true
+    }
+  }
+}
+
+function minimumAmount (amount) {
```
Comment (Tommytrg) [inline src/components/BetForm.vue]: Can you move this functions to methods?

Comment (ghost) [reply src/components/BetForm.vue]: Yes, but why don't you like it the way it is? You can think of these as utility functions that don't need the context of an object to be invoked.

**Pull request merged**